manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml

```diff
@@ -144,7 +144,16 @@ spec:
                 configuration: |
                   {
                     "debug": "false",
-                    "stat_prefix": "istio"
+                    "stat_prefix": "istio"{{- if .Values.global.multiCluster.clusterName }},
```

Comment:
Is this user facing? Should we make it `istio-clustername` if so?

Reply:
oh nevermind, you aren't changing the stat_prefix this is json being super annoying to template?

Reply:
yes. this is just about getting the comma in the correct location.